.circleci/config.yml

```diff
@@ -0,0 +1,106 @@
+# These environment variables must be set in CircleCI UI
+#
+# DOCKERHUB_REPO - docker hub repo, format: <username>/<repo>
+# DOCKER_USER - Login user for docker hub
+# DOCKER_PASS - Login password for docker hub user
+version: 2.0
+
+jobs:
+  release-tag: # build for the master branch and tags
+    machine: true
+    working_directory: ~/addons-frontend-code
```

Comment:
Is `addons-frontend-code` the name of the project we want to use? How about `addons-code-manager` instead?